README.md

```diff
@@ -1,146 +1,26 @@
-# Overview
+# [![avoCart](public/og-img-lg.png 'avoCart banner - Welcome to your smart shopping list!')](https://tcl-22-shopping-list.web.app/)
 
-## What is this project?
+[AvoCart](https://tcl-22-shopping-list.web.app/) is a smart shopping list that learns your buying habits and helps you remember what you need to buy on your next trip to the store.
 
-The goal of this project is to collaborate with your Collab Lab team to create a “smart” shopping list app that learns your buying habits and helps you remember what you’re likely to need to buy on your next trip to the store.
+Developed by participants of [The Collab Lab](https://the-collab-lab.codes/)'s Spring 2021 cohort, using React.js, Firebase, Material UI, and Styled Components
 
-## How does it work?
+## Demo
 
-As a user, you will enter items (e.g., “Greek yogurt” or “Paper towels”) into your list. Each time you buy the item, you mark it as purchased in the list. Over time, the app comes to understand the intervals at which you buy different items. If an item is likely to be due to be bought soon, it rises to the top of the shopping list.
```

Comment:
I like the use of video here. I think we should also keep the text description as an option to learn about the project without watching the video.

Reply:
Great idea! Added the "How does it work?" section back to the readme.